packages/sx.js/src/highlightConstants.ts

```diff
@@ -81,6 +81,18 @@ export const TOWNHALL_CONFIG = {
         { name: 'space', type: 'string' },
         { name: 'id', type: 'uint64' }
       ]
+    },
+    claimRole: {
+      ClaimRole: [
+        { name: 'space', type: 'string' },
```

Comment:
Shouldn't this be a number too?

Reply:
Space is what we put in URL: http://localhost:8080/#/townhall/ethereum/settings so there it's `ethereum`.